File: Javascript/205-isomorphic-strings.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
var isIsomorphic = function(s, t) {
    for(let i = 0; i < s.length; i++){
        const a = s.indexOf(s[i])
        const b = t.indexOf(t[i])
        
        if(s[i] !== s[b] || t[i] !== t[a]) return false
    }
    return true

};

File: Python/205-isomorphic-strings.py
#Solution
class Solution:
    def isIsomorphic(self, s: str, t: str) -> bool:
        mapS = {}
        mapT = {}

        for i in range(len(s)):
            if (s[i] in mapS and mapS[s[i]] != t[i]) or (t[i] in mapT and mapT[t[i]] != s[i]):
                return False
            mapS[s[i]] = t[i]
            mapT[t[i]] = s[i]
        
        return True